packages/types/src/primitive/sszTypes.ts

```diff
@@ -54,11 +57,21 @@ export const Wei = UintBn256;
 export const Root = new ByteVectorType(32);
 export const BlobIndex = UintNum64;
 
+export class ExecutionAddressType extends ByteVectorType {
+  constructor() {
+    super(20);
```

Comment:
As we have a custom type now makes sense to add typeName

```suggestion
    super(20, {typeName: "ExecutionAddress"});
```